packages/appium/lib/appium.js

```diff
@@ -202,9 +202,8 @@ class AppiumDriver extends DriverCore {
   }
 
   async getAppiumSessionCapabilities () {
```

Comment:
Maybe we can remove the method from the umbrella driver completely as we are not going to implement it there anyway

Reply:
Then, we should remove GET_APPIUM_SESSION_CAPABILITIES_COMMAND stuff as well, correct?
https://github.com/appium/appium/pull/20979/commits/f327d31a62835e34cd2317d5a1a43d21e6ce3d62

Reply:
I assume we must make a decision here whether we want this method to be implemented in each driver or in the umbrella one. I would still look for the umbrella one, since we have all the necessary properties known in the BaseDriver class, unless you need to put something special into the output, which we cannot unify